### ISSUE TITLE
(SOLUTION) OSX instructions and launch script differ: Arbites fails to launch

### ISSUE THREAD

**Issue by alexpw** [blahlicus/arbites-family#6]
TLDR
-
`ln -s /Library/Frameworks/Mono.framework/Commands/mono /usr/local/bin/mono`

Detail:
-
OSX instructions: https://github.com/blahlicus/arbites-family/wiki/Getting-Started#os-x
launch script: /Applications/Arbites.app/Contents/MacOS/Arbites

The Mono Framework install target is /Library/Frameworks/Mono.framework
The `mono` cli location is /Library/Frameworks/Mono.framework/Commands/mono
The launch script invokes /usr/local/bin/mono

Working Mono install options:
1. http://www.mono-project.com/download/
2. `brew cask install mono-mdk`

Both options install to the same place, neither creates a symlink in /usr/local/bin.

Installing with `brew install mono` does not produce a working environment, afaict, with it complaining about "Could not load file or assembly gtk-sharp" when trying to run `mono Arbites.exe`.  I abandoned that, brew uninstall mono, and used cask install with the symlink.

**Comment by leothelocust** [comment]
I added the symlink command to the Getting Started guide.
